src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/Converters/CustomDateTimeConverter.cs

```diff
@@ -8,8 +9,8 @@ public class CustomDateTimeConverter : JsonConverter<DateTime>
     {
         public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
-            var dateTime = DateTime.Parse(reader.GetString());
-            return DateTime.SpecifyKind(dateTime, DateTimeKind.Utc);
+            var parsedDateTime = DateTime.Parse(reader.GetString(), CultureInfo.InvariantCulture);
+            return DateTime.SpecifyKind(parsedDateTime.ToUniversalTime(), DateTimeKind.Utc);

```

Comment:
What does the `.ToUniversalTime()` do here? Isn't it enough to just specify the kind?

Reply:
> What does the `.ToUniversalTime()` do here? Isn't it enough to just specify the kind?

Otherwise the test fails since the datetime is not utc.

Reply:
@Liteolika If you add `DateTimeStyles.AdjustToUniversal` as a parameter on the `DateTime.Parse` method, you would get the same result. Without the need of specifying UTC kind and using `ToUniversalTime()`
Can you change this?

Reply:
@Liteolika can you please follow up on @zunkas's suggestion above?